contracts/infinite-pool/RewardsPoolBaseInfinite.sol

```diff
@@ -168,21 +165,6 @@ contract RewardsPoolBaseInfinite is Ownable {
         emit Started(startTimestamp, endTimestamp, rewardPerSecond);
     }
 
-    /** @dev Cancels the scheduled start. Can only be done before the start.
```

Comment:
Why is this removed?

Reply:
Following the discussion we had yesterday in the meeting, the cancel shouldn't be a thing in infinitePools, so I'll close this one @Mr-ONADASKY